src/core/tsi/ssl_transport_security.cc

```diff
@@ -1024,6 +1025,23 @@ static void tsi_ssl_handshaker_factory_init(
   gpr_ref_init(&factory->refcount, 1);
 }
 
+static char* ssl_get_peer_cert_chain_contents(
```

Comment:
Please add a unit test for this function.

Reply:
Can you suggest a file?

Reply:
There are some certificate files in https://github.com/grpc/grpc/tree/master/src/core/tsi/test_creds and feel free to add new ones (e.g., containing intermediate root certs).